src/main/webapp/pipe.js

```diff
@@ -472,7 +492,8 @@ function generateAggregatedChangelog(stageChanges, aggregatedChangesGroupingPatt
             var matches = stageChange.message.match(re) || [unmatchedChangesKey];
 
             Q.unique(matches).forEach(function (match) {
-                (changes[match] || (changes[match] = [])).push(stageChange);
```

Comment:
And here I have removed inline assignment. Similar to eslint no-cond-assign rule